src/js/components/Box/doc.js

```diff
@@ -317,11 +317,11 @@ of indicating the DOM tag via the 'as' property.`,
       ]),
       PropTypes.string,
     ]).description('A fixed width.'),
-    wrap: PropTypes.bool
-      .description(
-        `Whether children can wrap if they
-      can't all fit.`,
-      )
+    wrap: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.oneOf(['wrap', 'nowrap', 'wrap-reverse']),
+    ])
+      .description(`Controls the wrap property of the flexbox`)
```

Comment:
Our general pattern is not to describe properties by just describing the CSS property name. We've tried to not assume the caller understands the details of flexbox. For instance, our description of `alignSelf` says:
```
How to align along the cross axis when contained in a Box or along the column axis when contained in a Grid.
```

Also, we've tried to not use the CSS value where it seems awkward. For instance, for `alignSelf` we use `start` instead of `flex-start`, again, not requiring the user to think in pure CSS flexbox language. We've had some flexbox experts not like this approach but it has worked well for many users who aren't super familiar with flex-box. I say this to suggest that I think it would be more in line with the rest of the Box props if we had the value for `wrap` be:
```
PropTypes.oneOfType([PropTypes.bool, PropTypes.oneOf(['reverse'])])
```

Reply:
Ok makes sense. left the description as it was and added the reverse prop.